hack/make/binary-proxy

```diff
@@ -2,9 +2,16 @@
 
 set -e
 
+case "$(go env GOOS)/$(go env GOARCH)" in
+	windows/arm64) ;;
+		# TODO remove windows/arm64 when aarch64-w64-mingw32-gcc toolchain is available
```

Comment:
I don't think we need gcc for docker-proxy.
But windows/arm64 is a new golang target anyway.